.travis.yml

```diff
@@ -39,7 +39,7 @@ cache:
 
 script:
   - ./gradlew clean check
-  - ./gradlew -b toothpick-sample/build.gradle clean check -x lint
+  - ./gradlew -b toothpick-sample/build.gradle clean build -x lint
```

Comment:
why this change?

Reply:
Is this to prove that the build fails for the sample project?

Reply:
Yes, it is because `:build` fails on task `:mergeExtDexDebug`